documentation/modules/security/proc-renewing-ca-certs-manually.adoc

```diff
@@ -44,6 +46,29 @@ If maintenance time windows are configured, the Cluster Operator will generate t
 +
 Client applications must reload the cluster and clients CA certificates that were renewed by the Cluster Operator.
 
+. Check the period the CA certificate is valid:
++
+For example, using an `openssl` command:
++
+[source,shell,subs="+quotes"]
+kubectl get secret _CA-CERTIFICATE-SECRET_ -o 'jsonpath={.data._CA-CERTIFICATE_}' | base64 -d | openssl x509 -subject -issuer -startdate -enddate -noout
++
+_CA-CERTIFICATE-SECRET_ is the name of the `Secret`, which is `_KAFKA-CLUSTER-NAME_-cluster-ca-cert` for the cluster CA certificate and `_KAFKA-CLUSTER-NAME_-clients-ca-cert` for the clients CA certificate.
++
+_CA-CERTIFICATE_ is the name of the CA certificate, such as `jsonpath={.data.ca\.crt}`.
```

Comment:
So, the CA certificate in this example is named `ca\.crt`?

Reply:
I believe we need an escaped forward slash here. But I'll check.

Reply:
Great, thank you.

Reply:
@laidan6000 eacape is needed. :+1: